C/Cairo/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+name = "Cairo"
+version = v"1.14.12"
+
+sources = [
+    "https://www.cairographics.org/releases/cairo-1.14.12.tar.xz" =>
+    "8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/cairo-*/
+
+LDFLAGS="${LDFLAGS} -L${prefix}/lib"
+# TODO: find out why we need to manually add "${prefix}/include/freetype2",
+# the pkg-config file for freetype2 should already have this information.
+CFLAGS="-I${prefix}/include -I${prefix}/include/freetype2"
```

Comment:
Don't ask me why the build system can't get that it needs to use `-I${prefix}/include/freetype2`, it's in freetype2's pkg-config file :man_shrugging:  I went with the easy solution of just manually doing it.